### PR TITLE
change XDEBUG_CONFIG=remote_host for local dev

### PR DESCRIPTION
**Pull request by shakyShane**
For easier Xdebug setup.

Not sure if `templates/docker/env/production.env.dist` should be changed also?

### CODE REVIEW

File: templates/docker/env/local.env.dist [hunk 1]
```diff
@@ -30,12 +30,12 @@ MAIL_PORT=1025
 
 ## Xdebug config
 XDEBUG_IDE_KEY=PHPSTORM
-XDEBUG_CONFIG=remote_host=10.254.254.254
+XDEBUG_CONFIG=remote_host=docker.for.mac.host.internal
```
Comment (Loxzibit) [inline templates/docker/env/local.env.dist:33]: @shakyShane Is this the same as `docker.for.mac.localhost`

Comment (shakyShane) [reply templates/docker/env/local.env.dist:33]: works the same for now, but Docker advise against `docker.for.mac.localhost` now,

`docker.for.mac.host.internal` ftw

Comment (Loxzibit) [reply templates/docker/env/local.env.dist:33]: 👍

**Comment by mikeymike** [comment]
@shakyShane easy one for me thanks 👍

**Pull request merged**